### PR TITLE
Update root readme with November releases

### PR DESCRIPTION
**Pull request by kaerm**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -17,30 +17,30 @@ The client libraries are supported on Python 2.7 and 3.5.3 or later.
...
 
 
-### Client: July 2019 Preview
-New wave of packages that we are currently releasing in **Preview**. These libraries allow you to use and consume existing resources and interact with them, for example: upload a blob. These libraries share a number of core functionalities such as: retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure-core](./sdk/core/azure-core) library. You can learn more about these libraries by reading guidelines that they follow [here](https://azuresdkspecs.z5.web.core.windows.net/PythonSpec.html).
+### Client: November 2019 Releases
+New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries allow you to use and consume existing resources and interact with them, for example: upload a blob. These libraries share a number of core functionalities such as: retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure-core](./sdk/core/azure-core) library. You can learn more about these libraries by reading guidelines that they follow [here](https://azure.github.io/azure-sdk/python_introduction.html).
```
Comment (johanste) [inline README.md]: I'm good with merging this, but the term "GA" is a very Microsoft:y term. You generally have pre-release and stable releases.

Comment (johanste) [inline README.md]: While technically true, most of this functionality is exposed directly from the service-specific client libraries or from `azure-identity`. I would not link to `azure-core` from here. I'm on the fence as to if the python guidelines are appropriate as well. As a customer, I'm likely here to get stuff done. The design guidelines is something I might get to at some point. But not as the first thing I encounter.l

Comment (kaerm) [reply README.md]: 100% agreed, will have another wave of changes soon - for now just want to make sure we actually point to releases

**Pull request merged**